chevah/brink
# [#2992] Allow custom product name for download page.

Scope
---------

We now have a product name which is different than the base url name.

Changes
-----------

Allow a custom product name, other than the name used to construct the URL

How to test
--------------

reviewers: @alibotean 

this is just to let you know about the changes.

you can see the download page in the latest staging for MFT
